backend/web/test_settings.py

```diff
@@ -0,0 +1,9 @@
+from .settings import *
+
+
+DATABASES = {
```

Comment:
I was using SQLite in my settings.py where ENV='test' but yours is cleaner (and memory only) so I'm assuming we can remove this bit https://github.com/damianmoore/photo-manager/blob/master/backend/web/settings.py#L81